composer.json

```diff
@@ -15,12 +15,12 @@
     ],
     "require": {
         "php": ">=7.0",
-        "cakephp/cakephp": "^3.5"
+        "cakephp/cakephp": "^4.0"
     },
     "require-dev": {
-        "dereuromark/cakephp-ide-helper": "@stable",
-        "phpunit/phpunit": "^5.7.14|^6.0",
-        "cakephp/cakephp-codesniffer": "^3.0"
+        "phpunit/phpunit": "~8.5.0",
```

Comment:
I would keep `^` here - as it is unlikely the require-dev dependency here will create issues in minors.